### PR TITLE
Update RewardsParameters

### PR DESCRIPTION
**Pull request by thibault-martinez**
Replicate https://github.com/iotaledger/iota.go/pull/691

### CODE REVIEW

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 1]
```diff
@@ -139,17 +139,17 @@ interface RewardsParameters {
      */
     bootstrappingDuration: number;
     /**
-     * Mana Share Coefficient is the coefficient used for calculation of initial rewards.
+     * The rate of Mana rewards at the start of the bootstrapping phase.
      */
-    manaShareCoefficient: u64;
+    reward_to_generation_ratio: number;
```
Comment (DaughterOfMars) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: casing

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: fixed, thanks

File: bindings/nodejs/lib/types/models/info/node-info-protocol.ts [hunk 2]
```diff
@@ -139,17 +139,17 @@ interface RewardsParameters {
...
-     * Mana Share Coefficient is the coefficient used for calculation of initial rewards.
+     * The rate of Mana rewards at the start of the bootstrapping phase.
      */
-    manaShareCoefficient: u64;
+    rewardToGenerationRatio: number;
     /**
      * Decay Balancing Constant Exponent is the exponent used for calculation of the initial reward.
      */
-    decayBalancingConstantExponent: number;
+    initialTargetRewards_rate: u64;
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/types/models/info/node-info-protocol.ts]:
```suggestion
    initialTargetRewardsRate: u64;
```

Comment (thibault-martinez) [reply bindings/nodejs/lib/types/models/info/node-info-protocol.ts]: oh ffs

**Review by Thoralf-M — DISMISSED** [review]

**Review by DaughterOfMars — CHANGES_REQUESTED** [review]
python unused imports

**Review by DaughterOfMars — APPROVED** [review]
We have an issue to update the tests once the TIPs are?

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**